### PR TITLE
Fix datadog integration

### PR DESCRIPTION
**Pull request by wallies**

### CODE REVIEW

File: contrib-plugins/playbook.yml [hunk 1]
```diff
@@ -1,10 +1,14 @@
 # Sample plugins playbook.
 - hosts: all:!role=bastion
+
   tasks:
-    # https://github.com/DataDog/dd-agent/blob/master/conf.d/docker.yaml.example
```
Comment (enxebre) [inline contrib-plugins/playbook.yml]: don we stil need this?

File: terraform/aws/public-cloud/etcd_discovery_url.txt [hunk 2]
```diff
@@ -1 +1 @@
-https://discovery.etcd.io/fafb16ac572cae5895c6523e7b3bd640
\ No newline at end of file
+https://discovery.etcd.io/e141564df4f4f85905a0d9cb447c6c2d
```
Comment (enxebre) [inline terraform/aws/public-cloud/etcd_discovery_url.txt]: I guess we need to git update-index --assume-unchanged this file

**Comment by enxebre** [comment]
Otherwise than that looks good to me @wallies

File: contrib-plugins/playbook.yml (hunk 1 above)
Comment (wallies) [inline contrib-plugins/playbook.yml]: its in the ansible role, as we cant run pre_tasks until 2.0

**Pull request merged**